sdk/apis.go

```diff
@@ -6,60 +6,59 @@ import (
 	"github.com/celestiaorg/bittwister/api/v1"
 )
 
+const endpointPrefix = api.EndpointPrefix
+
 type PacketLossStartRequest = api.PacketLossStartRequest
 type BandwidthStartRequest = api.BandwidthStartRequest
 type LatencyStartRequest = api.LatencyStartRequest
 type ServiceStatus = api.ServiceStatus
 type MetaMessage = api.MetaMessage
 
 func (c *Client) PacketlossStart(req PacketLossStartRequest) error {
-	_, err := c.postResource("/packetloss/start", req)
-	return err
+	return c.postServiceAction(endpointPrefix+"/packetloss/start", req)
 }
 
 func (c *Client) PacketlossStop() error {
-	_, err := c.postResource("/packetloss/stop", nil)
-	return err
+	return c.postServiceAction(endpointPrefix+"/packetloss/stop", nil)
 }
 
 func (c *Client) PacketlossStatus() (*MetaMessage, error) {
-	return c.getServiceStatus("/packetloss/status")
+	return c.getServiceStatus(endpointPrefix + "/packetloss/status")
 }
 
 func (c *Client) BandwidthStart(req BandwidthStartRequest) error {
-	_, err := c.postResource("/bandwidth/start", req)
-	return err
+	return c.postServiceAction(endpointPrefix+"/bandwidth/start", req)
```

Comment:
[nit]: consistency across `endpointPrefix+` and `endpointPrefix +` is needed to be the same. fmt can help with this

Reply:
Thanks I'll fix this one and the endpoint paths will be resolved in another PR.